mautic/mautic
# Fixed several bugs that could cause contact import to fail

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | Y
| New feature? | 
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | #3010
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

This PR fixes several bugs with imports.

#### Steps to test this PR:
1. Create a custom multiselect field for a lead and set a default value.
2. Create a custom company field with a default value.
3. Unpublish a contact field with a default value.
4. Create a CSV file with those field values.
5. Import and it should work

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Before the PR, import that CSV file.
2. Note that the line will fail with an empty value (without the multi select value) or maybe with an error about the multi select value.
